db_schema/migrations/10_voting_agent.sql

```diff
@@ -0,0 +1,4 @@
+-- store the user agent as well to detect bot votes
+ALTER TABLE feedbackVote ADD COLUMN agent VARCHAR(255) NOT NULL AFTER user_hash;
```

Comment:
will it truncate if the user agent is longer?

Reply:
mysql does it in the worst case, yes. would be better to filter it earlier but this is also valid for all other params we take from outside.